scripts/github-changelog.sh

```diff
@@ -19,3 +19,5 @@ echo "- Generating changelog (${OUTPUT_FILE})"
 changelog-gen -changelog "${CHANGELOG_TEMPLATE}" \
 	-owner "${GH_ORG}" -repo "${GH_REPO}" \
 	"${RELEASE_SHA}" "${MASTER_SHA}" | tee "${OUTPUT_FILE}"
+
+echo "**Docker Image Perma Link**: ghcr.io/${GH_ORG}/${GH_REPO}:${RELEASE_SHA}" >> ${OUTPUT_FILE}
```

Comment:
Can we use the actual version tag in the link? The `${RELEASE_SHA}` would be very confusing _(should we want anything static, it should be docker image digest, but I think the release version makes most sense)._

Reply:
Also can we please add the docker link via the `changelog.tmpl`, instead of "echoing" to the generated file (the templating language has `env` functionality [^1], so we can pull it from env variable there).

[^1]: http://masterminds.github.io/sprig/os.html

Reply:
yeah sorry the docs for this where very hard to find, not great.